src/Honeybee_ExportEPC.py

```diff
@@ -127,18 +140,48 @@ def main(HBZones):
     # return surface data
     return srfData
     # done!
-
-
+#print srfData
 srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
 directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
 
+
```

Comment:
I think how you are doing the calculation in the lines below is fine but you are iterating through the surfaces two times which you don't really need to. You can easily add them to dictionary at line 132 and you won't need the code below. Let me know if you want me to provide some sample code.